Source/ViewableController.swift

```diff
@@ -291,13 +284,26 @@ class ViewableController: UIViewController {
     }
 
     func playAction() {
+        #if os(iOS)
         self.repeatButton.alpha = 0
         self.pauseButton.alpha = 0
         self.playButton.alpha = 0
         self.videoProgressView.alpha = 0
 
         self.videoView.play()
         self.requestToHideOverlayIfNeeded()
+        #else
+            // We use the native video player in Apple TV because it provides us extra functionality that are not
+            // providing in the custom player while at the same time it doesn't decrease the user experience since
+            // it's not expected that the user will drag the video to dismiss it, something that we need to do on iOS.
+            if let url = self.viewable?.url {
+                let controller = AVPlayerViewController(nibName: nil, bundle: nil)
```

Comment:
I'm not really folowing the structure here I believe. You are presenting a `AVPlayerViewController` over what kind of view exactly? doesnt that view contain a custom player itself? Than it feels a bit like a quick fix but that code-wise is not really logical.